src/util/entities/VoiceState.ts

```diff
@@ -84,8 +84,8 @@ export class VoiceState extends BaseClass {
 	@Column({ nullable: true })
 	self_stream?: boolean;
 
-	@Column()
-	self_video: boolean;
+	@Column({ nullable: true })
+	self_video?: boolean;
```

Comment:
why should this be nullable?

Reply:
Discord.js makes spacebar close out if it is not

Reply:
discords docs say otherwise 
![image](https://github.com/spacebarchat/server/assets/14828766/35098d2e-cb86-408e-8f0a-39c4f841d9f1)
https://discord.com/developers/docs/resources/voice#voice-state-object

Reply:
Ok, thats for the question mark, but the nullable?

Reply:
> Ok, thats for the question mark, but what about the nullable?

the docs dont say anything about it being nullable